app/models/mortgage_calculator/affordability.rb

```diff
@@ -23,8 +23,10 @@ def empty?
       empty
     end
 
-    delegate :committed_costs, to: :outgoings
-    delegate :fixed_costs, to: :outgoings
+    delegate :committed_costs,   to: :outgoings
+    delegate :fixed_costs,       to: :outgoings
+    delegate :rent_and_mortgage, to: :outgoings
```

Comment:
Can we join in one call to the method?
 e.g `delegate :committed_costs, :fixed_costs, :rent_and_mortgage, to: :outgoings `